### ISSUE TITLE
Training question

### ISSUE THREAD

**Issue by PhanThanhTrung** [luminxu/Pose-for-Everything#7]
Hi, I'm interested in your work and I'm trying to retrain it.
I see that you provide a command to train in multiple GPU with this:
```
./tools/dist_train.sh ${CONFIG_FILE} ${GPU_NUM} [optional arguments]
```
but I don't know what config file you mention in the command. Can you tell me more detail about it?

**Comment by PhanThanhTrung** [comment]
I have noticed that you use COCO format for the dataset. So I export Carfusion Dataset to COCO Format and training with only CarFusion. The MSE Loss is always at 0.003 and doesn't change during training time.

**Comment by luminxu** [comment]
In ``Usage``, we provide an example of the command that you mentioned. Please refer to it for more details.

For training loss, it should decrease until convergence (maybe around 0.003). Kindly note that training and testing on a single category is an easier setting than our proposed Category-Agnostic Pose Estimation (CAPE), which will make the model coverage faster. Please check ``acc_pose`` (that is similar to the metric PCK) to see whether the training process is normal.

**Comment by PhanThanhTrung** [comment]
@luminxu for the training part, I did train your method using CarFusion dataset, and it's okay. For the second part of your response, the mse loss during training is 0.003 and it doesn't change. By the way, I checked the acc_pose logged while training and it is about 0.33 😢😢😢
That's so weird... my assumption for this is because of the dataloader. I will verify it and note if there is a bug. Thank you for responding.

**Comment by tuanho27** [comment]
Hi Trung, just come across your issue. I think Carfusion seems tricky in the pre-process as my experiment, if you train on single category, I suggest checking the `num_queries` variable or using another dataset (ex: 300W, AFLW, etc) to test first cuz it's not too large. 
After a while play with the full data and make some small adjustments, here is my training log at epoch 3 for your reference.
![image](https://user-images.githubusercontent.com/22056468/185801784-3987734b-94e3-47e8-845e-d46039123bb2.png)
Hope it help

**Comment by liza-li** [comment]
@PhanThanhTrung 
Hi, could you please tell me how to train this model with only one dataset? I want to use only COCO dataset to train this model but I failed.  I would be so grateful for a response.
